app/views/shared/_card.html.erb

```diff
@@ -0,0 +1,24 @@
+<% @cars.each do |car| %>
+  <div class="col-xs-12 col-sm-6 col-md-4 ">
+    <div class="cards">
```

Comment:
why do we have a class cards inside the loop? Should it be outside the loop since its plural?